query04.sql

```diff
@@ -0,0 +1,34 @@
+/*
+Using the shapes.txt file from GTFS bus feed, find the two routes with the longest trips. In the final query, give the trip_headsign that corresponds to the shape_id of this route and the length of the trip.
+
+*/
+
+
+
+WITH copy_shapes AS (
+	select * 
+    FROM SEPTA_BUS_SHAPES)
+
+select 
+	shape_id as trip_headsign,
+	sum(st_distance(st_transform(geo_1,32129),st_transform(geo_2,32129))) as trip_length
```

Comment:
I like this solution too. `ST_MakeLine` was the solution that I was thinking of, but this produces an answer that's just a few meters off (I'm actually not sure why the answers from this `SUM` approach and the `ST_MakeLine` approach are slightly different, but the logic of your solution makes sense).